src/RenderWebGL.js

```diff
@@ -623,15 +623,19 @@ class RenderWebGL extends EventEmitter {
         const drawable = this._allDrawables[drawableID];
         if (!drawable) return null;
 
+        // Convert client coordinates into absolute scratch units
+        const scratchX = this._nativeSize[0] * ((x / this._gl.canvas.clientWidth) - 0.5);
+        const scratchY = this._nativeSize[1] * ((y / this._gl.canvas.clientHeight) - 0.5);
```

Comment:
A yeah there is a lot packed in there. `x` starts as the position of the click from the top left in the range `[0, clientWidth]`. Dividing by client width gets you to `[0, 1]`. The `-0.5` then centers the input from `[-0.5, 0.5]`. Then multiplying by the native size brings it to `[-0.5 * nativeSize, 0.5 * nativeSize]` (i.e. scratch coordinates!